### PR TITLE
Fix inputting a keysend on the send page

### PR DESCRIPTION
**Pull request by benalleng**
As mention in issue #200 I added the logic to handle p2pkh as an option for the payment type. 
As you can see in the commit history I'm still a green coder! lol

### CODE REVIEW

File: frontend/src/routes/Send.tsx [hunk 1]
```diff
@@ -93,6 +93,14 @@ function Send() {
       await navigateForInvoice(destination)
     } else if (paymentType === PaymentType.bip21) {
       await navigateForBip21(destination)
+    } else if (paymentType === PaymentType.keysend) {
+      if (sendAll === "true") {
+        const params = objectToSearchParams<SendConfirmParams>({ destination, all: "true" })
+        navigate(`/send/confirm?${params}`)
+      } else {
+        const params = objectToSearchParams<SendConfirmParams>({ destination })
+        navigate(`/send/amount?${params}`)
```
Comment (benthecarman) [inline frontend/src/routes/Send.tsx:102]: Doing send all for keysend is tricky and not something we currently support, can just exclude that do just use the `else` case here

**Comment by benthecarman** [comment]
Looks like your commit history got merged with #204

File: frontend/src/routes/Send.tsx (hunk 1 above)
Comment (benalleng) [reply frontend/src/routes/Send.tsx]: Ok! should i go ahead and remove the if statement all together or should it send some sort of UI for the user to understand they can't sendAll for keysend

**Review by benthecarman — APPROVED** [review]

File: frontend/src/routes/Send.tsx (hunk 1 above)
Comment (benthecarman) [reply frontend/src/routes/Send.tsx]: This looks good! The sendAll parameter is never exposed through the api so you should be okay.

**Comment by benthecarman** [comment]
Updated the title to be more correct. P2PKH is an onchain address type, whereas a key send is for sending to a lightning node.

**Review by TonyGiorgio — APPROVED** [review]
LGTM, though not sure if we want @futurepaul 's look at it first.

Also note to the PR merger, this should be squashed and then merged.

**Comment by benthecarman** [comment]
> sweet thank you... miss that exhaustive pattern matching in rust!

we could use a `switch` here instead

**Comment by futurepaul** [comment]
true but I think we'll want to do more cleanup / logic centralizing / refactoring in the future so this is good for now

**Pull request merged**